### PR TITLE
Test table creation error when PartitionKey or ClusteringKey have no Name attribute

### PR DESCRIPTION
**Pull request by landim**

### CODE REVIEW

File: src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs [hunk 1]
```diff
@@ -52,11 +48,11 @@ public void CreateTable_FluentMapping_Success()
         [Test]
         public void CreateTable_PartitionKeyOmitted()
         {
-            Map<ManyDataTypesPoco> mappingWithoutPk = new Map<ManyDataTypesPoco>();
-            var table = new Table<ManyDataTypesPoco>(_session, new MappingConfiguration().Define(mappingWithoutPk));
+            var mappingWithoutPk = new Map<ManyDataTypesPoco>();
+            var table = new Cassandra.Data.Linq.Table<ManyDataTypesPoco>(_session, new MappingConfiguration().Define(mappingWithoutPk));
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs]: Use namespace (that way is easier to merge on the dse driver)

File: src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs [hunk 2]
```diff
@@ -72,14 +68,14 @@ public void CreateTable_MakeAllPropertiesCaseSensitiveAtOnce()
                 .TableName("tbl_case_sens_once")
                 .CaseSensitive());
 
-            var table = new Table<ManyDataTypesPoco>(_session, config);
+            var table = new Cassandra.Data.Linq.Table<ManyDataTypesPoco>(_session, config);
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs]: Same.

File: src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs [hunk 3]
```diff
@@ -72,14 +68,14 @@ public void CreateTable_MakeAllPropertiesCaseSensitiveAtOnce()
...
             table.Create();
 
             var mapper = new Mapper(_session, config);
-            ManyDataTypesPoco manyTypesInstance = ManyDataTypesPoco.GetRandomInstance();
+            var manyTypesInstance = ManyDataTypesPoco.GetRandomInstance();
             mapper.Insert(manyTypesInstance);
-            string cqlSelect = string.Format("SELECT * from \"{0}\" where \"{1}\"='{2}'", table.Name, "StringType", manyTypesInstance.StringType);
-            List<ManyDataTypesPoco> objectsRetrieved = mapper.Fetch<ManyDataTypesPoco>(cqlSelect).ToList();
+            var cqlSelect = $"SELECT * from \"{table.Name}\" where \"{"StringType"}\"='{manyTypesInstance.StringType}'";
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs]: `\"{"StringType"}\"` within string interpolation can be simplified to the literal `\"StringType\"`.

File: src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs [hunk 4]
```diff
@@ -33,15 +29,15 @@ public override void OneTimeSetUp()
...
             table.Create();
 
             var mapper = new Mapper(_session, mappingConfig);
-            ManyDataTypesPoco manyTypesInstance = ManyDataTypesPoco.GetRandomInstance();
+            var manyTypesInstance = ManyDataTypesPoco.GetRandomInstance();
 
             mapper.Insert(manyTypesInstance);
-            string cqlSelect = string.Format("SELECT * from \"{0}\" where \"{1}\"='{2}'", table.Name, "StringType", manyTypesInstance.StringType);
-            List<ManyDataTypesPoco> instancesQueried = mapper.Fetch<ManyDataTypesPoco>(cqlSelect).ToList();
+            var cqlSelect = $"SELECT * from \"{table.Name}\" where \"{"StringType"}\"='{manyTypesInstance.StringType}'";
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs]: Can be simplified into `\"StringType\"=...`

File: src/Cassandra.IntegrationTests/Mapping/Tests/Attributes.cs [hunk 5]
```diff
@@ -362,14 +360,14 @@ public void Attributes_CompositeKey()
...
         public void Attributes_MultipleClusteringKeys()
         {
-            MappingConfiguration config = new MappingConfiguration();
+            var config = new MappingConfiguration();
             config.MapperFactory.PocoDataFactory.AddDefinitionDefault(typeof(PocoWithClusteringKeys), () => LinqAttributeBasedTypeDefinition.DetermineAttributes(typeof(PocoWithClusteringKeys)));
             var table = new Table<PocoWithClusteringKeys>(_session, config);
             table.Create();
 
             var cqlClient = new Mapper(_session, config);
 ;
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Mapping/Tests/Attributes.cs]: If you don't mind fixing this one :)

File: src/Cassandra.IntegrationTests/Mapping/Tests/Attributes.cs [hunk 6]
```diff
@@ -56,32 +53,32 @@ public void Attributes_Ignore_TableCreatedWithMappingAttributes()
...
             var mapper = new Mapper(_session, new MappingConfiguration());
-            PocoWithIgnoredAttributes pocoToUpload = new PocoWithIgnoredAttributes
+            var pocoToUpload = new PocoWithIgnoredAttributes
             {
                 SomePartitionKey = Guid.NewGuid().ToString(),
                 IgnoredStringAttribute = Guid.NewGuid().ToString(),
             };
             mapper.Insert(pocoToUpload);
-            string cqlSelect = string.Format("SELECT * from \"{0}\" where \"{1}\"='{2}'", table.Name.ToLower(), "somepartitionkey", pocoToUpload.SomePartitionKey);
+            var cqlSelect = $"SELECT * from \"{table.Name.ToLower()}\" where \"{"somepartitionkey"}\"='{pocoToUpload.SomePartitionKey}'";
```
Comment (jorgebay) [inline src/Cassandra.IntegrationTests/Mapping/Tests/Attributes.cs]: Can be simplified into `\"somepartitionkey\"=...`

**Comment by landim** [comment]
Addressed the suggestions:
- Fixed string interpolations
- Usage of namespace alias to avoid conflicts on Classes names (from Mapping and Linq)

**Pull request merged**